### DISCUSSION TITLE
Menambahkan variabel jam (hh:mm:ss)

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#111] [Q&A] [answered]
Bagaimana caranya untuk menambahkan 1 variabel yang isinya hanya waktunya saja dengan format hh:mm:ss dari data terlampir
<img width="131" alt="image" src="https://user-images.githubusercontent.com/103427989/168491408-a4976d97-534e-4546-b3fa-cb462f7d9be2.png">

**Comment by VictorNugraha** [comment, marked as answer]
Halo, @joejnt!

Untuk menambahkan 1 kolom yang berisikan data waktu, kita dapat memanfaatkan fungsi **_format()_**. Berikut contoh codenya,

```
format(x = data$nama_kolom, # nama kolom yang inigin diextract
       format = "%H:%M:%S") # format yang diharapkan  
```

Demikian informasi yang dapat saya sampaikan, semoga menjawab pertanyaan yang ditanyakan. Selamat mencoba :D

**Reply by onlyphantom** [reply]
@VictorNugraha i made a small edit (`data#nama_kolom` -> `data$nama_kolom`) hope you don't mind!

**Reply by VictorNugraha** [reply]
@onlyphantom Thank you Sam!

**Reply by joejnt** [reply]
Thank you for the insight
